api/v1alpha1/status.go

```diff
@@ -124,6 +124,13 @@ type RuntimeStatus struct {
 	// MountTime represents time last mount happened
 	// if Mounttime is earlier than master starting time, remount will be required
 	MountTime *metav1.Time `json:"mountTime,omitempty"`
+
+	// MountPoints represents the mount points specified in the bounded dataset
+	DatasetMounts []DatasetMount `json:"datasetMounts,omitempty"`
+}
+
+type DatasetMount struct {
```

Comment:
Why not using 

```
// the info of mount points have been mounted
	Mounts []Mount `json:"mounts,omitempty"`
```

Reply:
I'm not sure if it's okay to add too many properties in Runtime's status, especially for now the properties are only useful for ThinRuntime.

Reply:
can we just exclude option and EncryptOptions?

Reply:
Done.